src/reducers/states.ts

```diff
@@ -120,16 +128,25 @@ export const defaultApplication: IApplicationState = {
   environment: currentEnvironment()
 }
 
-export const defaultPlayground: IPlaygroundState = {
+export const defaultPlayground: IPlaygroundState = {}
+
+export const createDefaultWorkspace: () => IWorkspaceState = () => ({
   context: createContext(latestSourceChapter),
-  editorValue: '',
+  editorValue: undefined,
   editorWidth: '50%',
   isRunning: false,
   output: [],
   replValue: '',
   sideContentActiveTab: 0,
-  sideContentHeight: undefined,
+  sideContentHeight: 60,
```

Comment:
Does this cause overflow? A fresh workspace should render such that the side content takes up a size that is just big enough to prevent overflow of its contents.